pkgs/tools/misc/calamares-nixos-extensions/default.nix

```diff
@@ -5,10 +5,10 @@ stdenv.mkDerivation rec {
   version = "0.3.16";
```

Comment:
Should be `0.3.16-unstable-YYYY-MM-DD` (why do we even tag that repo?).

Reply:
I don't think we'll merge this until the nixos-extensions repo has been merged. Sure, tagging is maybe not the most relevant on it, but I'll keep with the tradition for now, and update the version here when we have a new tag.

Reply:
I just mean, since this isn’t actually the tagged revision, our standards say that it should be suffixed with `-unstable-` and the date stamp of the latest commit.

Reply:
(And it’s not entirely just a stylistic nitpick, because it makes it a lot easier to determine whether a package is the fixed version or not.)

Reply:
@emilazy Right, and if we were going to merge this without a new tag on calamares-nixos-extensions, I'd agree. But my plan is to create a new tag on this repo and update this PR to point to it before removing the "draft" status from this PR.

Reply:
Oh, okay. Never mind me, then.